odoo/addons/base/views/res_users_views.xml

```diff
@@ -520,6 +541,11 @@ if records:
                                     <button name="preference_change_password" type="object" string="Change password" class="btn btn-secondary"/>
                                 </div>
                             </group>
+                            <group name="access" string="Other Devices">
+                                <div colspan="2">
+                                    <button name="preference_revoke_all_devices" type="object" string="Log out from all devices" class="btn btn-secondary"/>
```

Comment:
Note that generally action button calls method beginning with "action" (action_revoke_all_devices), just to know this is an UI business method.